containers/app/Dockerfile

```diff
@@ -81,8 +81,6 @@ RUN python opendevin/core/download.py # No-op to download assets
 # opendevin:opendevin -> opendevin:app
 RUN find /app \! -group app -exec chgrp app {} +
 
-RUN chown -R opendevin:app /app/logs && chmod -R 770 /app/logs # This gets created by the download.py script
```

Comment:
sure about this?

Reply:
Yep - just build the `app` container and tested locally - seems to work!

Reply:
ok, was surprised seeing a Dockerfile change in this PR, just wanted to make sure :)